makefiles/localnet.mk

```diff
@@ -4,20 +4,28 @@
 
 # TODO_TECHDEBT(@olshansk): Look at `make dev_up` in the `path` repo and port it here.
 .PHONY: localnet_up
-localnet_up: check_pocketd warn_message_grove_helm_charts check_docker_ps check_kind_context proto_regen localnet_regenesis ## Starts up a clean localnet
+localnet_up: check_pocketd check_docker_ps proto_regen localnet_regenesis dev_up check_kind_context warn_message_acc_initialize_pubkeys ## Starts up a clean localnet
 	tilt up
 
 .PHONY: localnet_up_quick
-localnet_up_quick: check_docker_ps check_kind_context ## Starts up a localnet without regenerating fixtures
+localnet_up_quick: check_docker_ps dev_up check_kind_context ## Starts up a localnet without regenerating fixtures
 	tilt up
 
 .PHONY: localnet_down
 localnet_down: ## Delete resources created by localnet
 	tilt down
 
+.PHONY: dev_up
+dev_up: check_kind # Internal helper: Spins up Kind cluster if it doesn't already exist
+	echo "[INFO] Creating kind cluster..."; \
+	kind create cluster --config ./localnet/kubernetes/kind-config.yaml; \
+	kubectl config use-context kind-kind; \
+	kubectl create namespace path; \
+	kubectl create namespace monitoring; \
+	kubectl create namespace middleware; \
```

Comment:
Should we remove the trailing `\` here?

I feel like it should theoretically break things, no?